.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v5.0.0
     with:
       license-check: true
+      node-versions: '["16", "18", "20", "22"]'
```

Comment:
Should we test on node 16?

Reply:
also on v14, I don't think we have shipped a major or this yet.

Reply:
We might need tape, will take a look